src/app/content/search/components/SearchResultsSidebar/SearchResultsBarWrapper.tsx

```diff
@@ -125,6 +125,12 @@ export class SearchResultsBarWrapper extends Component<ResultsSidebarProps> {
 
   public componentDidUpdate() {
     this.scrollToSelectedPage();
+
+    const searchSidebar = this.searchSidebar.current;
+
+    if (!searchSidebar) {
+      return;
+    }
```

Comment:
Will this prevent anything? Since it is in `componentDidUpdate` everything should be already updated so this is most likelly not preventing anything from happening